client/src/components/TeamTotal.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const TeamTotal = (props) => (
+  <div className="team-total-container">
+    <div className="team-total">
+      <h3>Number of Current Team <span>Members</span> <a href="#">{props.total}</a></h3>
```

Comment:
I'm not used to seeing "props" written as anything but this.props, making it "this.props.total". Would you agree this is good practice?

Reply:
As I understand it, components using ES6 class refer to props as this.props.
If you write the component as an arrow function props is just props, this binding is different in an arrow function.